openedx/features/wikimedia_features/messenger/static/messenger/js/hooks/useInboxList.js

```diff
@@ -1,9 +1,10 @@
 import { useState, useEffect } from 'react';
 import useClient from "./useClient"
-import { ToastsStore } from "react-toasts";
+
+import { toast } from 'react-toastify';
 
 export default function useInboxList(inboxPageNumber, setSelectedInboxUser, context) {
-    const { client } = useClient()
+    const { client, notification } = useClient()
     const [inboxList, setInboxList] = useState([]);
     const [inboxHasMore, setInboxHasMore] = useState(false);
```

Comment:
Kindly, try to keep things consistent. 
We're using `;` on some lines and for some lines we're skipping it.